exercises/run-length-encoding/RunLengthEncodingTests.elm

```diff
@@ -29,17 +33,24 @@ tests =
           (decode "12WB12W3B24WB")
         )
     , test
-        "decode(encode(...)) combination"
+        "(decode (encode (...)) combination"
```

Comment:
Good catch here. Personally I like to have this kind of housekeeping in a separate commit. It tends to add to the noise when doing code reviews (I'm a bit of a stickler about this in my day job but not so much in OS PRs). You can just commit and merge in yourself without review I would think.